pkg/cidr/cidr_test.go

```diff
@@ -183,7 +172,9 @@ func (t *CidrTestSuite) TestEqual(c *check.C) {
 		},
 	}
 	for _, tt := range tests {
-		c.Assert(tt.fields.n.Equal(tt.args.o), check.Equals, tt.want, check.Commentf("Test Name: %s", tt.name))
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
Not sure if I'd create subtests here, their names are completely meaningless. You're already passing tt.name as the last argument to `require.Equal`, so it's already clear which assertion failed.

Reply:
As we are changing to require for assertion, this is to make sure that all the tests are run. Not to mention that we can have a capability to run one individual test.